Subida de imágenes a AWS

rojas1995 opened the pull request:
La funcionalidad está hecha, pedidme las keys para probarla. Las fotos que ya estaban en la BBDD no se muestran, hay que subir esas fotos a AWS y cambiarlas en el populate. Cuando se actualiza la foto de perfil o de un evento, entonces se mustran bien

Commit:
[feat] Desde el django admin se pueden subir fotos

Desde el django admin si sube las fotos a AWS pero desde la app normal no. Está llegando al save del form con el campo picture vacío, no sé muy bien por qué. Además una vez están subidas no se muestran.
Para hacer pruebas no va la cuenta de AWS student, dice que no reconoce las keys, así que para probar las cosas hay que usar las keys buenas.

raunavcru committed:
[fix] Files

raunavcru committed:
[feat] Se muestra foto

raunavcru committed:
[feat] event

raunavcru committed:
Update list_search.html

raunavcru committed:
[feat] Function de limite

Commit:
[fix] He cambiado el detail.html por el que me ha dado juanmi

Para resolver los conflictos que había

rojas1995 committed:
Merge branch 'dev' into feat/be/M-10-alternative

angbongon commented:
> La funcionalidad está hecha, pedidme las keys para probarla. Las fotos que ya estaban en la BBDD no se muestran, hay que subir esas fotos a AWS y cambiarlas en el populate. Cuando se actualiza la foto de perfil o de un evento, entonces se mustran bien

Vale Rojas. Lo revisamos esta tarde.

Commit:
[fix] Actualizo el requirements

Commit:
[feat][refactor] Definido backend de storage y adaptación populate

El populate ahora funciona con la implementación de AWS

Commit:
[fix] Imagen de usuario centinela

juanmiguelruiz commented:
Cuando sergio arregle el css avisadme y lo reviso

Commit:
[cfg] Eliminado warning al usar AWS

sergiodpino committed:
[fix] Arreglo boton subir imagen

sergiodpino committed:
Merge branch 'feat/be/M-10-alternative' of https://github.com/eventshow/eventshow into feat/be/M-10-alternative

sergiodpino committed:
[fix] Arreglos bugs sonarcloud

sergiodpd commented:
El del profile/update no se ve del todo bien pero es por un cambio que tengo que hacer en el front-end, lo haré en otra rama si os parece bien.

angbongon committed:
Merge branch 'dev' into feat/be/M-10-alternative

Commit:
[fix] Imagen de perfil se muestra cuando se usa movil

Commit:
[fix] Imagen de deleted añadida a media para prueba local

También está subida al bucket de S3. Se ha cambiado la ruta donde se
encuentra dicha imagen.

angelcalzadollamas committed:
[fix] Ahora sale error si falta imagen

angelcalzadollamas committed:
[fix] El mensaje de error del campo "sobre mí" ahora hace referencia a ese campo

angelcalzadollamas committed:
[fix] En el filtro se controla que el min price no sea negativo

angelcalzadollamas committed:
Merge branch 'dev' of https://github.com/eventshow/eventshow into dev

Commit:
[WIP] Validación en backend de tamaño de imagen y tipo

El tipo de imagen se valida sólo cuando se crea un nuevo evento, o se
añade por primera vez una imagen al perfil

angbongon committed:
Merge branch 'dev' into feat/be/M-10-alternative
